Competition2017/src/competition/subsystems/shooter_wheel/commands/UnjamShooterWheelCommand.java

```diff
@@ -0,0 +1,33 @@
+package competition.subsystems.shooter_wheel.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.shooter_wheel.ShooterWheelsManagerSubsystem;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class UnjamShooterWheelCommand extends BaseShooterWheelCommand {
```

Comment:
Did we have instances of balls jammed in the wheels? I only recall agitator/belts